docs/site/Order-filter.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Order filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, order
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Order-filter.html
+summary: An <i>order</i> filter specifies a set of logical conditions to match, similar to a ORDER clause in a SQL query.
+---
+
+An _order_ filter specifies how to sort the results: ascending (ASC) or descending (DESC) based on the specified property.
+
+### REST API
+
+Order by one property: 
+
+<pre>
+?filter[order]=<i>propertyName<i>%20<ASC|DESC>
+</pre>
+
+Order by two or more properties:
+
+<pre>
+?filter[order][0]=<i>propertyName</i> <ASC|DESC>&filter[order][1]=<i>propertyName</i> <ASC|DESC>...
+</pre>
+
+Where:
+
+- _propertyName_ is the name of the property (field) to sort by. 
+- `<ASC|DESC>` signifies either ASC for ascending order or DESC for descending order.
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
```

Comment:
Let's show a example of URL with stringified JSON format for the `order` filter.